index.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const recurse = require('reftools/lib/recurse.js').recurse;
-const clone = require('reftools/lib/clone.js').clone;
+const clone = require('reftools/lib/clone.js').deepClone;
```

Comment:
Why the need to change the clone strategy?

Reply:
Ah sorry didn't mean to commit this line. Revert!